src/Utils/MockData/BackendMock.js

```diff
@@ -0,0 +1,51 @@
+import Config from './../../config/config.js';
+import Hierarchy from './Transport for London/Hierarchy.js';
+import Entity from './Transport for London/Entity.js';
+import EntityType from './Transport for London/EntityType.js';
+
+class BackendMock {
+
+	static getRouteMapper() {
+		return new Map()
+			.set(/^entitytype\/hierarchy$/i, BackendMock.getEntityTypeHierarchy)
+			.set(/^entitytype\/(-?\d+)\/attributes$/i, BackendMock.getEntityTypeAttributes)
+			.set(/^entity\/(-?\d+)\/children\/type\/(-?\d+)\/(((\w)+)\+)*(\w)*$/i, BackendMock.getChildEntitiesOfEntityType)
+			.set(/^entity\/(-?\d+)$/i, BackendMock.getEntity);
+	}
+		
+	static handleRequest(request) {
+		const RouteMapper = BackendMock.getRouteMapper();
+		const cleanedRoute = request.url.replace(Config.backendRESTRoute,"");
+		const targetRoute = [...RouteMapper.keys()].find((route) => {
```

Comment:
`.keys()` gives you an iterator for _RouteMapper_. But `find()` expects an array. The `...` operator _converts_ the iterator to an array we can operate on.